components/Loading/Spinner.jsx

```diff
@@ -1 +1,3 @@
-export default () => <div><i className='fa fa-spinner fa-spin'></i></div>;
+import React from 'react';
```

Comment:
Should we call this directory "Spinner"?  I like having it match the component name